src/context/WalletProvider/WalletProvider.tsx

```diff
@@ -651,7 +696,9 @@ export const WalletProvider = ({ children }: { children: React.ReactNode }): JSX
   const handleAccountsOrChainChanged = useCallback(async () => {
     if (!walletType || !state.adapters) return
 
-    const localWallet = await state.adapters.get(walletType)?.pairDevice()
+    const keymanager = state.adapters.get(walletType)
```

Comment:
you can 1 line this everywhere

`    const localWallet = await state.adapters.get(walletType)?.[0]?.pairDevice()`